### DISCUSSION TITLE
intro

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#18] [Announcements]
# intro

include(@/info.snippet.md) 特性 [x] 快到极致的响应速度，Lighthouse 接近满分。; [x] 独一份的按需全自动 HTTPS，甚至不用填域名。; [x] 包括完整的前后台和服务端。; [x] 前台和后台都为响应式设计，完美适配移动端和多尺寸设备。; [x] 前台和后台都支持黑暗模式，并可自动切换。; [x] 前台为...

https://vanblog.mereith.com/intro.html

**Comment by qulishen** [comment]
这个项目能部署到githubpages上吗，没有钱买服务器，有教程吗？感谢！

**Reply by sunshang-hl** [reply]
确实需要这个功能,同求.

**Reply by xuxuxu1124** [reply]
这个有后台系统呀，GithubPages只适合静态的网站

**Comment by dupenf** [comment]
士大夫的撒

**Comment by happmaoo** [comment]
看见了

**Comment by xxddccaa** [comment]
你说的痛点都是我找项目的过程中遇到的痛点，没想到有这么好的项目，好像不更新了啊？
我搭建的：https://dong-blog.fun/

**Comment by JinPengGeng** [comment]
好像不更新了啊？
